CMakeLists.txt

```diff
@@ -219,7 +219,7 @@ endif(${WITH_XFS})
 
 option(WITH_SPDK "Enable SPDK" OFF)
 if(WITH_SPDK)
-  find_package(dpdk REQUIRED)
+  find_package(dpdk 16.7 REQUIRED)
```

Comment:
@yuyuyu101 it's a follow up of https://github.com/ceph/ceph/commit/96046f53e9072cee8ce332392635669593335be2#commitcomment-20417415 . 

actually, i tend to put `find_package(dpdk 2.0 REQUIRED)` instead to be consistent with https://github.com/ceph/ceph/pull/12857/commits/1844b729663164248bca17db3e220e036910841a#diff-3b85bae027325c1a3d65ac2b5d316239L32 .